scopes/workspace/variants/variants.docs.mdx

```diff
@@ -1,8 +1,8 @@
 ---
-description: teambit.workspace/variants enables cascading configurations by selection of component groupings or sets in the workspace and applying configurations to these cascading groups
+description: Enables the configuration of group of Components.
```

Comment:
Configuration variants allow to configure a selected group of components.

Reply:
done